### DISCUSSION TITLE
Changing default speed of navigate_to function in Python

### DISCUSSION THREAD

**Discussion by sleeplessTLV** [iRobotEducation/create3_docs#370] [Questions]
Hi All,
we'd like to make the robot move more slowly whan it navigates to a new spot. However we could not find a way to do it.
Any ideas?
thanks

**Comment by shamlian** [comment]
Assuming you're using ROS 2, you'll see the robot takes a maximum translation and rotation speed as arguments. See the [message which defines the action](https://github.com/iRobotEducation/irobot_create_msgs/blob/rolling/action/NavigateToPosition.action) for details. There is no analog using the Python SDK, though there might be some tricky way if I think hard enough.
